kubernetes/charts/edge-kubernetes/values.yaml

```diff
@@ -167,6 +167,9 @@ edgeAgent:
     # false dy default.
     # runAsNonRoot: false
 
+    # If an environment variables is required and not defined above, they may be added as needed.
+    # ANOTHER_ENV_VAR: "another env value"
```

Comment:
Does this mean we allow customer to define their own environment variable?

Reply:
Yes. We have added parameters for all known useful values, but there may be something we have missed, or some variable we hadn't considered the user wanting to set, like "CoolOffTimeUnitInSeconds" which can be modified, but rarely used.